carousel.js

```diff
@@ -0,0 +1,62 @@
+
+
+class Carousel {
+	constructor(element) {
+		this.element = element;
+		this.arrows = element.querySelectorAll(".Arrows");
+		this.counter = 0;
+		this.pages = new Pages(this.element);
```

Comment:
You don't want to be passing "this.element" into Pages, but rather do a query selector here the pages class element, and pass that into Pages.